docs/user-guide/configure-certificates.md

```diff
@@ -113,3 +113,46 @@ This error has to be resolved before you can proceed with the next installation
 **Notes:** 
 
 - On many z/OS systems, the certificate for z/OSMF is not signed by a trusted CA and is a self-signed certificate by the z/OS system programmer who configured z/OSMF.  If that is the case, then Zowe itself will not trust the z/OSMF certificate and any function dependent on z/OSMF will not operate correctly.  To ensure that Zowe trusts a z/OSMF self-signed certificate, you must use the value `VERIFY_CERTIFICATES=false` in the `zowe-setup-certificates.env` file.  This is also required if the certificate is from a recognized CA but for a different host name, which can occur when a trusted certificate is copied from one source and reused within a z/OS installation for different servers other than that it was originally created for.  
+
+## Configure Zowe Certificates with z/OSMF Workflows
+
+z/OSMF Workflow lets you create a keystore directory that is used by Zowe to hold the certificate that is used for encrypting communication between Zowe clients and the Zowe z/OS servers.
+
+Perform the following steps to register and execute the Zowe workflow in the z/OSMF web interface:
+
+ 1.	Log in to the z/OSMF web interface.
+ 2.	Select **Workflows** from the navigation tree.
+ 3.	Select Create Workflow from the **Actions** menu.
+ 4.	Enter the complete path to the workflow definition file in the **Workflow Definition filed**.
+    The path to the workflow definition file is <extracted_pax_folder>/files/workflows/ ZWEWRF05.xml file. 
```

Comment:
`workflows/ ZWEWRF05.xml file`

Extra space before `ZWEWRF05.xml`. It has several occurrences.